shared/components/Conference/index.js

```diff
@@ -1,28 +1,5 @@
 import React from 'react';
-import SiteFooter from '../SiteFooter';
-import Hero from '../Hero';
-import RedBadgerBanner from '../RedBadgerBanner';
-import MailingList from '../MailingList';
-import ConferenceAbout from '../ConferenceAbout';
-import NextConferenceEvent from '../NextConferenceEvent';
-import ShareYourIdeas from '../ShareYourIdeas';
-
-const first = {
```

Comment:
Just removed so people can view the component. Don't merge it though because I am going to put these back in